packages/sit-onyx/src/styles/grid.scss

```diff
@@ -1,102 +1,96 @@
-//
-// Default Grid Variants
-//
-
 // prettier-ignore
 $gridVariants: (
-//  NAME     BREAKPOINT  COLUMNS  GUTTER  MARGIN
-    "2xs"    none        4        1rem    1rem,
-    "xs"     577px       8        1rem    1rem,
-    "sm"     769px       8        1.5rem  2rem,
-    "md"     993px       12       1.5rem  4rem,
-    "lg"     1441px      16       2rem    4rem,
-    "xl"     1921px      16       2rem    4rem,
+//  NAME     BREAKPOINT  MAX-VARIANT  COLUMNS  GUTTER  MARGIN
+    "2xs"    none        none         4        1rem    1rem,
+    "xs"     577px       none         8        1rem    1rem,
+    "sm"     769px       none         8        1.5rem  2rem,
+    "md"     993px       none         12       1.5rem  4rem,
+    "lg"     1441px      "md"         16       2rem    4rem,
+    "xl"     1921px      "lg"         16       2rem    4rem,
 );
 
+/**
+ * Grid container base class
+ */
 .onyx-grid {
   display: grid;
   grid-template-columns: repeat(var(--onyx-grid-columns), minmax(0, 1fr));
   gap: var(--onyx-grid-gutter);
-  margin: var(--onyx-grid-margin);
+  // Its called margin in the UX concept. So we keep that name for consistency, even when it is implemented through a padding.
+  padding: var(--onyx-grid-margin);
```

Comment:
🤨 
If its a padding, can't we adjust the UX concept accordingly to also call it by what it is?^^

Reply:
Alternatively implement it as margin, is there a reason why its padding?

Reply:
We use "margin: 0 auto" for centering and the padding to ensure the Grid-Margin is still applied.
Afaik "padding" is not a concept that is used or exists in UX